DirNIA/readme.md

```diff
@@ -0,0 +1,87 @@
+# The Summer school project (TutorAssistant)
+
+## Description:
```

Comment:
Very nice job! 🔥
Can we please add a small "How to use" section where we explain how to quickly run the app? Thanks 🙏🏼